x-pack/plugins/observability_solution/uptime/server/legacy_uptime/lib/alerts/common.ts

```diff
@@ -16,7 +16,7 @@ import { RuleExecutorServices } from '@kbn/alerting-plugin/server';
 import { addSpaceIdToPath } from '@kbn/spaces-plugin/common';
 import { AlertInstanceState } from '@kbn/alerting-plugin/server';
 import { AlertInstanceContext } from '@kbn/alerting-plugin/server';
-import { uptimeRuleFieldMap } from '../../../../common/rules/uptime_rule_field_map';
+import { syntheticsRuleFieldMap } from '@kbn/synthetics-plugin/common/rules/synthetics_rule_field_map';
```

Comment:
I want to thoroughly avoid changes to Uptime. I feel like any kind of unnecessary change raises the risk matrix for having to deal with Uptime SDHs, and we certainly don't have time for that. I don't think this is a necessary change.

Reply:
this should be reverted, will do it.